package.json

```diff
@@ -8,11 +8,11 @@
     "format:write": "yarn prettier --write .",
     "lint": "echo 'Linting all packages' && wsrun --parallel --exclude-missing lint",
     "test": "echo 'Running tests' && wsrun --parallel --exclude-missing test",
-    "build": "echo 'Building all packages' && wsrun --stages --exclude-missing build",
+    "build-common": "echo 'Building all packages' && wsrun --stages --exclude-missing build-pre-app",
```

Comment:
What is build-pre-app? Looks like a dupe of build

Reply:
this is to decoupling common packages build from all builds ....

Reply:
Why two names for the same thing? `common`/`pre-app`